src/applications/edu-benefits/tests/1995/containers/StemEligibilityView.unit.spec.jsx

```diff
@@ -0,0 +1,193 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { expect } from 'chai';
+
+import StemEligibilityView from '../../../1995/containers/StemEligibilityView';
+import createCommonStore from '../../../../../platform/startup/store';
```

Comment:
Use the platform alias here to shorten the path.

Reply:
shortened